Probesys/bileto
# new: Allow to attach labels to tickets

## Related issue(s)



https://github.com/Probesys/bileto/issues/283

## How to test manually



1. Make sure your user has the permission to create labels and update tickets' labels
2. Create at least 2 labels in the admin
3. Create a ticket and select one label
4. Verify the ticket is created and it has the label
5. Update the ticket's label by unselecting the previous one and selecting the other one
6. Verify the ticket's labels are updated and the change appears in the ticket's timeline

## Checklist
